testsuite/features/reposync/srv_add_rocky8_repositories.feature

```diff
@@ -50,6 +50,11 @@ Feature: Add the Rocky 8 distribution custom repositories
     And I click on "Save Repositories"
     Then I should see a "repository information was successfully updated" text
 
+  Scenario: Synchronize the repositories in the custom channel for Rocky 8 DVD
+    When I call spacewalk-repo-sync for channel "rocky-8-iso"
```

Comment:
The method above, where we can pass a custom url, it's about the custom repo?
Maybe we can generate both repos plus sync the channel all in one command line execution? 🤔

Reply:
In any case, LGTM, you can iterate over it if you want later.

Reply:
There was a method with a custom URL. 

```ruby
When(/^I call spacewalk\-repo\-sync for channel "(.*?)" with a custom url "(.*?)"$/) do |arg1, arg2|
  @command_output, _code = $server.run("spacewalk-repo-sync -c #{arg1} -u #{arg2}", check_errors: false)
end
```

Reply:
Yup, the method above the one I marked :)
I'm suggesting to refactor it to support multiple custom repos.